io.github.atari800.Atari800.yml

```diff
@@ -0,0 +1,54 @@
+app-id: io.github.atari800.Atari800
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: atari800
+build-options:
+  no-debuginfo: true
+  strip: true
+modules:
+  - name: libsdl
+    sources:
+      - type: archive
+        url: https://libsdl.org/release/SDL-1.2.15.tar.gz
+        sha256: d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00
+      - type: patch
+        path: patch/sdlpatch.txt
```

Comment:
patch is missing.

Reply:
Instructions said to only commit the manifest.  Will commit the rest.